index.html

```diff
@@ -0,0 +1,487 @@
+                    <li>Arts & Design</li>
+                    <li>Books</li>
+                    <li>Dance</li>
+                    <li>Movies</li>
+                    <li>Music</li>
+                    <li>Pop Culture</li>
+                    <li>Television</li>
+                    <li>Theater</li>
+                    <li>Video: Arts</li>
+                </ul>
+            </div>
+            <div class="living float-left">
+                <ul>
+                    <li><strong>LIVING</strong></li>
+                    <li>Automobiles</li>
+                    <li>Crossword</li>
+                    <li>Education</li>
+                    <li>Food</li>
+                    <li>Health</li>
+                    <li>Jobs</li>
+                    <li>Magazine</li>
+                    <li>Parenting</li>
+                    <li>Real Estate</li>
+                    <li>Style</li>
+                    <li>T Magazine</li>
+                    <li>Travel</li>
+                    <li>Weddings</li>
+                </ul>
+            </div>
+            <div class="more float-left">
+                <ul>
+                    <li><strong>MORE</strong></li>
+                    <li>Reader Center</li>
+                    <li>Wirecutter</li>
+                    <li>Live Events</li>
+                    <li>The Learning Network</li>
+                    <li>Tools & Services</li>
+                    <li>N.Y.C. Events Guide</li>
+                    <li>Multimedia</li>
+                    <li>Photography</li>
+                    <li>Video</li>
+                    <li>Newsletters</li>
+                    <li>NYT Store</li>
+                    <li>Times Journeys</li>
+                    <li>Manage My Account</li>
+                </ul>
+                </div>
+            </div>
+            <div class="subscribe float-right">
+                <ul>
+                    <li><strong>SUBSCRIBE</strong></li>
+                    <li class="logolinks"><strong><img src="./Assets/delivery.jpg" alt="Delivery">Home Delivery</strong></li>
+                    <li class="logolinks"><strong><img src="./Assets/digital.jpg" alt="Digital">Digital Subscriptions</strong></li>
+                    <li class="logolinks"><strong><img src="./Assets/crossword.jpg" alt="Crossword">Crossword</strong></li>
+                    <li class="logolinks"><strong><img src="./Assets/cooking.jpg" alt="Cooking">Cooking</strong></li>
+                    
+                   
+                    <li><a href="#">Email Newsletters</a></li>
+                    <li><a href="#">Corporate Subscriptions</a></li>
+                    <li><a href="#">Education Rate</a></li>
+                </ul>
+                <hr>
+                <ul>
+                    <li><a href="#">Mobile Applications</a></li>
+                    <li><a href="#">Replica Edition</a></li>
+                   
+                </ul>
+            </div>
+        </div>
+        <nav class="nav fixed top">
```

Comment:
Why did you placed the navbar tag almost in the end of your HTML code? The navbar must be inside the header tag, on the top of the document. Kindly change the place of the navbar.